File: tracker/src/test/java/testhelpers/TestActivity.java
package testhelpers;

import android.app.Activity;
import android.os.Bundle;

public class TestActivity extends Activity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle(getTestTitle());
    }

    public static String getTestTitle(){
        return "Test Activity";
    }
}

File: tracker/src/test/java/testhelpers/TestActivity.kt
package testhelpers

import android.app.Activity
import android.os.Bundle

class TestActivity : Activity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        title = testTitle
    }

    companion object {
        @JvmStatic
        val testTitle: String
            get() = "Test Activity"
    }
}
